lib/created_message_queue_handler.ts

```diff
@@ -276,6 +311,21 @@ export async function handleMessage(
   //  Email notification
   //
 
+  // check if the user has blocked emails sent from this service
+  // 'some(true)' in case we must send the notification by email
+  // 'none' in case the user has blocked the email channel
+  const maybeIsUnblockedEmail = fromPredicate(
+    (hasBlockedEmail: boolean) => !hasBlockedEmail
+  )(blockedChannels.has(BlockedChannelEnum.EMAIL));
+
+  if (isNone(maybeIsUnblockedEmail)) {
```

Comment:
try to avoid logic that is there only for logging purposes, perhaps you can move the logging to a code branch that gets executed only when `maybeIsUnblockedEmail` is empty

Reply:
it's used afterwards